CheckYourEligibility-Parent/Controllers/HomeController.cs

```diff
@@ -37,5 +56,48 @@ public IActionResult Parental_Guidance()
         {
             return View();
         }
+
+        [HttpGet]
+        public async Task<IActionResult> SchoolList()
+        {
+            var schoolList = await _parentService.GetSchool("school");
+
+            // Check if schoolList or schoolList.Data is null
+            var schools = schoolList?.Data?.ToList() ?? new List<CheckYourEligibility.Domain.Responses.Establishment>();
+
+            var viewModel = new SchoolListViewModel
+            {
+                Schools = schools,
+                IsRadioSelected = null // Default value to ensure radio buttons are blank
```

Comment:
isn't null default value?

Reply:
so would it be null if not specified when a new instance of the view model is being created?